website/docs/feature-flag-tutorials/django/django-examples.md

```diff
@@ -374,7 +374,7 @@ response = requests.request("GET", url, headers=headers, data=payload)
 print(response.text)
 ```
 
-Read our documentation on [Event logs](/reference/event-log) and [APIs](/reference/api/unleash/get-events-for-toggle) to learn more.
+Read our documentation on [Event logs](/reference/events#event-log) and [APIs](/reference/api/unleash/get-events-for-toggle) to learn more.
```

Comment:
I know it's out of scope for this PR but maybe we should try to make this consistent at a later point: "Event log" instead of "event logs"? 🤔

Reply:
50ee051a61e300c67373de15007ff3a3527a15fb